src/main/webapp/scripts/account-info.js

```diff
@@ -1,22 +1,40 @@
-let loggedInStatus;
-let logInOutURL;
-let signupURL;
-let userInfo;
+let auth2;
+let profile;
+const initPromise = init();
 
 /**
- * Updates global variables related to account data.
- * @return {Promise} after setting global variables
+ * Initializes the auth2 variable
+ * @return {Promise} Promise which resolves when auth2 is initalized
  */
-function updateAccountData() {
-  return fetch('/account').then((response) => response.json()).then(
-      (loginData) => {
-        loggedInStatus = loginData.loggedIn;
-        logInOutURL = loginData.logInOutURL;
-        signupURL = loginData.signUpURL;
-        if (loggedInStatus) {
-          userInfo = loginData.userInfo;
-        }
-      });
+function init() {
+  return new Promise(function(resolve, reject) {
+    gapi.load('auth2', function() {
```

Comment:
what is gapi again?

Reply:
google api loader. It's created from the script call in index.html